packages/bitcore-node/package.json

```diff
@@ -78,15 +78,15 @@
   "dependencies": {
     "JSONStream": "~1.3.1",
     "abi-decoder": "2.0.1",
-    "bitcore-client": "8.7.3",
-    "bitcore-lib": "8.7.3",
-    "bitcore-lib-cash": "8.7.3",
-    "bitcore-p2p": "8.7.3",
-    "bitcore-p2p-cash": "8.7.3",
-    "bitcore-wallet-client": "8.7.3",
+    "bitcore-client": "^8.8.0",
+    "bitcore-lib": "^8.8.0",
+    "bitcore-lib-cash": "^8.8.0",
+    "bitcore-p2p": "^8.8.0",
+    "bitcore-p2p-cash": "^8.8.0",
+    "bitcore-wallet-client": "^8.8.0",
```

Comment:
bitcore-wallet-client is not being used by bitcore-node

Reply:
this PR only update the version, we can remove it on a following one.